unstable/surface-layers.xml

```diff
@@ -81,9 +239,11 @@
         depths. The compositor may choose under which circumstances to send
         inputs. By convention, the compositor will send input events to
         the top-most surface of the top-most layer.
+
+        Interactivity is double-buffered, see wl_surface.commit.
       </description>
-      <arg name="input_types" type="uint" summary="mask of input devices to use"/>
-      <arg name="exclusive_types" type="uint" summary="mask of input devices to use exclusively"/>
+      <arg name="input_types" type="uint" enum="wl_seat::capability" summary="mask of input devices to use"/>
+      <arg name="exclusive_types" type="uint" enum="wl_seat::capability" summary="mask of input devices to use exclusively"/>
```

Comment:
https://github.com/wayland-project/wayland-protocols/blob/master/unstable/tablet/tablet-unstable-v2.xml It looks like new input types won't be added to the `wl_seat::capability`, so this approach likely won't work to catch all input devices in the future

Reply:
Not sure how we should handle input types from other protocols in this protocol. Providing our own enum isn't really a solution.